server/models/newsSchema.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose');
+var moment = require('moment');
+
+const Schema = mongoose.Schema;
+
+const newsSchema = new Schema(
+    { 
+        event: {type: String, enum: [ 'New Alumni', 'New Student', 'Confirmed Meeting' ] }, 
+        alumni: [{type: Schema.Types.ObjectId, ref: 'Alumni'}], 
+        students: [{type: Schema.Types.ObjectId, ref: 'Student'}], 
+        school: {type: Schema.Types.ObjectId, ref: 'School'},
+        dateCreated: {type: Date, default: moment()}, 
+        role: {type: String, default: 'BOTH', enum: ['BOTH', 'ALUMNI', 'STUDENT']},
+        timeElapsed: {type: String, required: false},
+        grade: {type: Number, default: null}
```

Comment:
Let's make this required `false`

Reply:
for grade? I use null as way to store posts that don't require grade info so that I can use null as an accepted value

Reply:
Yep! Wouldn't setting it to be `required: false` also only allow us to use it when needed?

Reply:
Nvm, this works!